src/pages/Lending/hooks/useAllLendingPositionsData.tsx

```diff
@@ -115,7 +115,23 @@ export const useAllLendingPositionsData = ({ assetId }: UseAllLendingPositionsDa
   )
 
   const isLoading = useMemo(() => positions.some(position => position.isLoading), [positions])
-  const isActive = useMemo(() => positions.some(position => position.data), [positions])
+
+  const isActive = useMemo(() => {
+    return positions.some(position => {
+      const data = position.data
+      if (!data) return false
+      const {
+        collateralBalanceCryptoPrecision,
+        collateralBalanceFiatUserCurrency,
+        debtBalanceFiatUserCurrency,
+      } = data
+      return (
+        (collateralBalanceCryptoPrecision && parseFloat(collateralBalanceCryptoPrecision) > 0) ||
+        (collateralBalanceFiatUserCurrency && parseFloat(collateralBalanceFiatUserCurrency) > 0) ||
+        (debtBalanceFiatUserCurrency && parseFloat(debtBalanceFiatUserCurrency) > 0)
```

Comment:
Think we're safe using only one of the two first as a criteria if we want to keep things simpler (no need to check for fiat if we check for crypto balance), though just checking for the debt should also be enough - if debt is repaid, there is no loan

Reply:
Yeah fair enough, simplified in https://github.com/shapeshift/web/pull/7792/commits/4de0861b62fface20bf0cdbf22f415cc7c5c296f 🙏